controller/src/enhancements/bomb.rs

```diff
@@ -148,29 +190,61 @@ impl BombInfo {
     }
 }
 
-/// % of the screens height
-const PLAYER_AVATAR_TOP_OFFSET: f32 = 0.004;
-
-/// % of the screens height
-const PLAYER_AVATAR_SIZE: f32 = 0.05;
-
 impl Enhancement for BombInfo {
-    fn update(&mut self, ctx: &crate::UpdateContext) -> anyhow::Result<()> {
+    fn update(&mut self, ctx: &UpdateContext) -> anyhow::Result<()> {
         if !ctx.settings.bomb_timer {
             return Ok(());
         }
 
+        for entity_identity in ctx.cs2_entities.all_identities() {
+            let entity_class = ctx
+                .class_name_cache
+                .lookup(&entity_identity.entity_class_info()?)?;
+
+            if !entity_class
+                .map(|name| *name == "C_CSPlayerPawn")
+                .unwrap_or(false)
+            {
+                /* entity is not a player pawn */
+                continue;
+            }
+
+            let player_pawn = entity_identity
+                .entity_ptr::<C_CSPlayerPawn>()?
+                .read_schema()
+                .unwrap();
+
+            let player_has_defuser = player_pawn
+                .m_pItemServices()?
+                .cast::<CCSPlayer_ItemServices>()
+                .reference_schema()?
+                .m_bHasDefuser()?;
+
+            if player_has_defuser {
+                self.num_kit += 1;
+            }
+        }
+
+        let local_controller = ctx.cs2_entities.get_local_player_controller()?;
+
+        if local_controller.is_null()? {
+            return Ok(());
+        }
+
+        let local_pawn = ctx
+            .cs2_entities
+            .get_by_handle(&local_controller.reference_schema()?.m_hPlayerPawn()?)?
+            .context("missing local player pawn")?
+            .entity()?
+            .read_schema()?;
+
+        self.local_team = local_pawn.m_iTeamNum().unwrap_or_default();
```

Comment:
I'm not entirely sure, but from my experience using `m_iPendingTeamNum` is more reliable then using `m_iTeamNum`. Especially after the half time, `m_iTeamNum` seems to be off for the first round.